guides/common/modules/proc_retrieving-live-metrics-using-cli.adoc

```diff
@@ -1,19 +1,24 @@
 [id='retrieving-live-metrics-using-cli_{context}']
-= Retrieving Live Metrics using CLI
+= Retrieving Live Metrics in the CLI
 
-To output metrics on disk partition write instances, enter the following command:
+You can use the PCP CLI tools to retrieve live metrics.
+
+* To print current values of a particular metric, enter the following command:
++
 ----
 # pmval -f 1 disk.partitions.write
 ----
-
-In this example, PCP converts the number of writes to disk partitions from a counter value, to a rate value.
-The `-f 1` specifies that the value be abbreviated to one decimal place.
-
-.Example output
++
+In this example, metric instances on writes to disk partitions are displayed.
+PCP converts the number of writes to disk partitions from a counter value to a rate value.
+The `-f 1` argument specifies to abbreviate the values to one decimal place.
++
+Example output:
++
 [options="nowrap", subs="verbatim,quotes,attributes"]
 ----
 metric:    disk.partitions.write
-host:      {foreman-example-com}
+host:      _{foreman-example-com}_
```

Comment:
Since this is example output, it can stay non-italicized, but I don't care either way.